scripts/build_bsp.sh

```diff
@@ -235,6 +249,11 @@ kernel_dir=$3
     make CROSS_COMPILE=aarch64-none-linux-gnu- ARCH=arm64 KERNELDIR=${kernel_dir} RGX_BVNC="33.15.11.3" BUILD=release PVR_BUILD_DIR=${pvr_target} WINDOW_SYSTEM=${pvr_window_system} &>>"${LOG_FILE}"
 
     log "ti-img-rogue-driver: installing .."
-    cd binary_am62_linux_wayland_release/target_aarch64/kbuild
-    make -C ${kernel_dir} ARCH=arm64 CROSS_COMPILE=aarch64-none-linux-gnu- INSTALL_MOD_PATH=${rootfs_dir} INSTALL_MOD_STRIP=1 M=`pwd` modules_install &>>"${LOG_FILE}"
+    if [ ${machine} = "j721s2-evm" ]; then
+        cd binary_j721s2_linux_wayland_release/target_aarch64/kbuild
```

Comment:
This can just be `binary_${pvr_target}_${pvr_window_system}_release` instead of having if statements for all supported machines.